src/Mul.tsx

```diff
@@ -0,0 +1,42 @@
+import { ChangeEvent, useState } from "react";
+import "./Mul.css";
+
+function Mul() {
+    const [numState, numSetState] = useState<number>(2); // 2부터 시작
+
+    const dan = [2, 3, 4, 5, 6, 7, 8, 9]; // 2단부터 9단까지
+    const number = [1, 2, 3, 4, 5, 6, 7, 8, 9];
```

Comment:
```suggestion
const dan = [2, 3, 4, 5, 6, 7, 8, 9]; // 2단부터 9단까지
const number = [1, 2, 3, 4, 5, 6, 7, 8, 9];

function Mul() {
    const [numState, numSetState] = useState<number>(2); // 2부터 시작
```

이렇게 바뀌지 않는 값, 즉 상수는 컴포넌트 밖에 선언하면 다시 생성되는 일이 없겠죠?!

Reply:
🙄😋👍